e2e_tests/test_suites/ospolicies/ospolicies_test_data.go

```diff
@@ -1399,6 +1399,50 @@ func buildLinuxExecResourceTests(name, image, pkgManager, key string) *osPolicyT
 									},
 								},
 							},
+							{
+								Id: "exec-output",
+								ResourceType: &osconfigpb.OSPolicy_Resource_Exec{
+									Exec: &osconfigpb.OSPolicy_Resource_ExecResource{
+										Validate: &osconfigpb.OSPolicy_Resource_ExecResource_Exec{
+											Source: &osconfigpb.OSPolicy_Resource_ExecResource_Exec_Script{
+												Script: "if ls $1 >/dev/null; then\nexit 100\nfi\nexit 101",
```

Comment:
Do you expect 101 will ever be returned? If not, would `exit 100` be clearer?

Reply:
101 is returned during check state, then enforce runs, then 100 is returned on post check state.